docs/user-guide/org-settings.md

```diff
@@ -2,9 +2,9 @@
 
 The Org Settings page is only available to organization admins. It can be found in the main navigation menu.
 
-## Org Information
+## General
 
-This tab lets you change the organization's name. This name must be unique.
+This tab lets you change the organization's name and URL identifier. Both of these must not be in use by any other orgs.
```

Comment:
I like that this is reworded @Shrinks99 I think we could go a step further in editing this, since there's no way for users to know what the other orgs are currently, and so we don't give them a success path to pair with this warning. Something like:

`Change your organization's name and URL identifier in this tab. Choose an org name that's unique and memorable, like the name of your company or organization. Org name and URLs are unique to each Browsertrix instance (for example, on browsertrix.com) and you may be asked to change the org name if either are already in use by another org.`

Also, documentation best practices usually calls for an active voice, which we employ for the most part. Instead of "This tab lets you ..." I believe we should opt for a more humanist and user-centered "Change org name ...".

Reply:
Updated here: https://github.com/webrecorder/browsertrix/pull/2020/commits/27361672dcb79c68d9e0427fb18e877e12b5a165